lib/Connector/XiboAudienceReportingConnector.php

```diff
@@ -280,6 +283,7 @@ public function onRegularMaintenance(MaintenanceRegularEvent $event)
             if (count($rows) > 0) {
                 try {
                     $response = $this->getClient()->post($this->getServiceUrl() . '/audience/receiveStats', [
+                        'timeout' => 180,
```

Comment:
I don't think we can have that request timing out - it needs to complete before progressing so that we're sure of the state in both systems.

As this runs in regular maintenance, we can set the timeout to be as long as we need, although I'd suggest we have to find a balance between the number of records we send and the amount of time we wait.